README.md

```diff
@@ -0,0 +1,199 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="soyMarco.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metric Web App <a name="about-project"></a>
+
+<div align="center">
+
+  <img src="stock_stats.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+**Metric Web App** is a project to show API data on a SPA.
+![Video](https://cdn.loom.com/sessions/thumbnails/dec4f57c759b400e8c4e7f14cb1df71e-with-play.gif)
```

Comment:
- [ ]  You did a great job updating the presentation videos, but it seems like you haven't addressed the suggestions made by the previous reviewer. To clarify their request, when they mentioned highlighting the portion of the code, it means that you should show the code on your videos as well. This entails sharing your screen and displaying the parts of the code that you are proud of working on. I hope this feedback doesn't discourage you. Perhaps it was a minor misunderstanding, but you can definitely do it. Just include that part in your videos, and you'll be ready to go.
![image](https://github.com/MarcoDDM/react-capstone/assets/43877689/1dca54cd-51f4-4021-8ac3-eabebe218f31)